src/Icicle/Sea/FromAvalanche/Psv.hs

```diff
@@ -828,163 +851,132 @@ seaOfWriteOutput ps oname@(OutputName name) otype0 ts0 ixStart
     hasDateTime (StructT m) = any hasDateTime $ Map.elems $ getStructType m
     hasDateTime _           = False
 
-seaOfOutput
-  :: Doc -> OutputName -> ValType -> [ValType] -> Int
-  -> Doc                                        -- buffer to print to
-  -> Either SeaError (Doc, [(BufSize, BufLen)]) -- name, alloc size and len
-
-seaOfOutput ps oname@(OutputName name) otype0 ts0 ixStart dstbuf
+seaOfOutput :: Doc -> OutputName -> ValType -> [ValType] -> Int -> Either SeaError Doc
+seaOfOutput ps oname@(OutputName name) otype0 ts0 ixStart
```

Comment:
yeah, ok, I'll need to do something like that for better outputting.

Gonna merge this and do that in another PR